addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -829,6 +829,10 @@ def drop_off_picked(self, continue_batch, move_line_ids, location_barcode, resul
                 picks_todo |= pick_todo
                 pick.write({"u_reserved_pallet": False})
 
+                # If we dropped a package then reset the last package reserved as
+                # it will not be possible to keep using it
+                self.u_last_reserved_pallet_name = False
```

Comment:
Is the update inside the loop?

Reply:
Yep, should be outside to be correct, fixing up soon.